airbyte-integrations/connectors/source-greenhouse/unit_tests/test_streams.py

```diff
@@ -135,29 +138,17 @@ def test_parse_response_expected_response(applications_stream):
         ]
     """
     response._content = response_content
-    parsed_response = applications_stream.parse_response(response)
+    parsed_response = applications_stream.retriever.parse_response(response, stream_state={})
     records = [record for record in parsed_response]
 
-    assert isinstance(parsed_response, GeneratorType)
     assert records == json.loads(response_content)
 
 
 def test_parse_response_empty_content(applications_stream):
     response = requests.Response()
+    response.status_code = 200
     response._content = b"[]"
-    parsed_response = applications_stream.parse_response(response)
+    parsed_response = applications_stream.retriever.parse_response(response, stream_state={})
     records = [record for record in parsed_response]
 
-    assert isinstance(parsed_response, GeneratorType)
     assert records == []
-
-
-def test_parse_response_invalid_content(applications_stream):
```

Comment:
this is not currently handled by low-code, but I'm not sure if
1. this ever happens
2. Ignoring bad responses is always the right thing to do (probably not)

Reply:
this test was created as part of the very first PR introducing this connector. So I"m not sure why it's here. Fine by me to remove.